src/main/java/org/folio/circulation/resources/renewal/RenewalResource.java

```diff
@@ -535,10 +546,7 @@ private Result<RenewalContext> validateIfRenewIsAllowed(RenewalContext context,
       final var errors = isDueDateRequired
         ? validateIfRenewIsAllowedAndDueDateRequired(loan, requestQueue)
         : validateIfRenewIsAllowedAndDueDateNotRequired(loan, requestQueue);
-      final var loanPolicy = loan.getLoanPolicy();
```

Comment:
Do not really need to remove it for the fix, but this seems to be some stale part left after refactorings. If "errors" list is not empty, validation failure is returned anyway in next lines. If "errors" is empty, validation failure with empty list is  returned and filtered out later by OverridingErrorHandler, so it does not affect the result response anyway, so removed it.